src/app/api/addusers/route.ts

```diff
@@ -0,0 +1,30 @@
+import { User } from "@/app/utils/interfaces";
+import { NextResponse } from "next/server";
+import { getUserCollection } from "../mongodb";
+
+export async function POST(req: Request) {
+  const body = await req.json();
+  const users = body.members as User[];
```

Comment:
Replace this with fetching members.

You could also possibly filter out users that are not active.